### PR TITLE
Js  > ts

### PR DESCRIPTION
**Pull request by webber-jiang**
Ignore the comments in IssuesReducer.ts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,10 +27,11 @@
     "eject": "react-scripts eject",
     "storybook": "start-storybook",
     "pre-commit": "lint-staged",
-    "build-storybook": "build-storybook -c .storybook -o .out"
+    "build-storybook": "build-storybook -c .storybook -o .out",
+    "eslint": "eslint . --ext .tsx,.ts"
   },
   "lint-staged": {
-    "src/**/*.{js,jsx,css, scss}": [
+    "src/**/*.{css, scss, ts, tsx}": [
```
Comment (Ray2126) [inline package.json:34]: You can't eslint css files.

**Comment by webber-jiang** [comment]
> There's only 1 problem with the eslint and css files but I change this to stylelint in my PR so I think it's fine if you merge.

**Comment by webber-jiang** [comment]
> There's only 1 problem with the eslint and css files but I change this to stylelint in my PR so I think it's fine if you merge.

OK

File: package.json (hunk 1 above)
Comment (webber-jiang) [reply package.json:34]: You changed it to style lint in other PR so it's fine.

**Pull request merged**